src/main/resources/application.yml

```
@@ -0,0 +1,40 @@
+  port: 8080
+
+spring:
+  application:
+    name: happy-pill
+  datasource:
+    driver-class-name: org.postgresql.Driver
+  jpa:
+    open-in-view: false
+
+
+--- #dev
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)